### PR TITLE
Fix some commands in the delete-cli.md

### PR DESCRIPTION
**Pull request by nksma**
Fix make argument for deleting GCP resource
Fix kubectl command to switch context.

### CODE REVIEW

**Comment by nksma** [comment]
@googlebot I signed it!

**Review by Arhell — COMMENTED** [review]
/assign @RFMVasconcelos

**Review by RFMVasconcelos — COMMENTED** [review]
Since this is GKE specific, I'd rather have someone from Google review this change. 

@Bobgy @joeliedtke could you PTAL?

**Comment by Bobgy** [comment]
/assign @zijianjoy 
Now owns kubeflow on GCP

File: content/en/docs/distributions/gke/deploy/delete-cli.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ This guide assumes the following settings:
 
     ```bash
     cd "${KF_DIR}"
-    make delete
+    make delete-gcp
```
Comment (zijianjoy) [inline content/en/docs/distributions/gke/deploy/delete-cli.md:36]:
```suggestion
    make delete
```

**Review by zijianjoy — COMMENTED** [review]
Thank you @nksma! It looks good to me except one place.

See https://github.com/kubeflow/gcp-blueprints/blob/master/kubeflow/Makefile#L73.

/lgtm

File: content/en/docs/distributions/gke/deploy/delete-cli.md (hunk 1 above)
Comment (nksma) [reply content/en/docs/distributions/gke/deploy/delete-cli.md]: Thanks, I didn't realize it was changed recently.

**Comment by Bobgy** [comment]
thanks!
/lgtm
/approve

**Pull request merged**